src/ng/directive/form.js

```diff
@@ -282,9 +288,24 @@ FormController.prototype = {
  * @name form.FormController#$setValidity
  *
  * @description
- * Sets the validity of a form control.
- *
- * This method will also propagate to parent forms.
+ * Change the validity state of the form, and notify the parent form (if any).
+ *
+ * Application developers will rarely need to call this method directly. It is used internally, by
+ * {@link ngModel.NgModelController#$setValidity NgModelController.$setValidity()}, to propagate a
+ * control's validity state to parent `FormControllers`.
```

Comment:
I would say "to the parent `FormController`" because it currently sounds a bit like a control can have multiple direct parent FormControllers